<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailChangeableClass</class>
 <widget class="QWidget" name="DetailChangeableClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加人员</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>工号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEmployeeNoChangeable">
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_10">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_14">
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>薪水</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineSalaryChangeable">
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>身份证号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineIdChangeable">
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_15">
    <item>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>职位</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="linePostChangeable">
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>180</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>部门</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineDepartmentChangeable">
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineNameChangeable">
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_17">
    <item>
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>地址</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineAddressChangeable">
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_18">
    <item>
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>出生日期</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineYearChangeable">
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>年</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineMonthChangeable">
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>月</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineDayChangeable">
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>日</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonCalendar">
      <property name="text">
       <string>选择</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>344</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_19">
    <item>
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>手机号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="linePhoneNoChangeable">
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_20">
    <item>
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>性别</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboSex">
      <item>
       <property name="text">
        <string>男</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>女</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_11">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_21">
    <property name="spacing">
     <number>40</number>
    </property>
    <item>
     <widget class="QPushButton" name="buttonOK">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCalendarWidget" name="calendarBirthday">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>248</width>
     <height>183</height>
    </rect>
   </property>
   <property name="gridVisible">
    <bool>false</bool>
   </property>
   <property name="dateEditEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
